tests/rust/tcp-test/close/wait.rs

```diff
@@ -224,3 +227,119 @@ fn wait_after_async_close_connecting_socket(libos: &mut LibOS, remote: &SocketAd
 
     Ok(())
 }
+
+fn wait_on_invalid_queue_token_returns_einval(libos: &mut LibOS) -> Result<()> {
+    println!("{}", stringify!(wait_on_invalid_queue_token_returns_einval));
+
+    // Wait on an invalid queue token.
+    match libos.wait(QToken::from(u64::MAX), Some(Duration::from_micros(0))) {
+        Ok(_) => anyhow::bail!("wait() should not succeed on invalid token"),
+        Err(e) if e.errno == libc::EINVAL => {},
+        Err(_) => anyhow::bail!("wait() should not fail with any other reason than invalid token"),
+    }
+
+    Ok(())
+}
+
+fn wait_for_accept_after_issuing_async_close(libos: &mut LibOS, local: &SocketAddrV4) -> Result<()> {
```

Comment:
Could you add a one-line comment about this test here? You may want to copy what we have for description on the issue.

Reply:
Done.